libexec/bats-core/bats

```diff
@@ -101,6 +101,7 @@ export BATS_TMPDIR="${TMPDIR:-/tmp}"
 BATS_TMPDIR=${BATS_TMPDIR%/} # chop off trailing / to avoid duplication
 export BATS_RUN_TMPDIR=
 export BATS_GUARANTEED_MINIMUM_VERSION=0.0.0
+export BATS_LIB_PATH=${BATS_LIB_PATH-/usr/lib/bats}
```

Comment:
Sorry for the drive-by comment, but shouldn't this be `${BATS_LIB_PATH:-/usr/lib/bats}`? (Note the colon.)

Reply:
I only translated this from the original Code but I assumed the intention is to allow for users to specify an empty search path. This would mit ve possible with colon.

Reply:
Sure, but the syntax for that is `${var:-default}`, not `${var-default}`, right? ([Bash docs](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html))

Reply:
No, :- is alternative value for unset or empty, - only replaces unset variables.

Reply:
I did not know that. Thank you for the explanation!